dxfeed/core/utils/handler.pyx

```diff
@@ -102,7 +103,11 @@ cdef class DefaultHandler(EventHandler):
 
 
         df = pd.DataFrame(df_data, columns=self.columns)
-        time_columns = df.columns[df.columns.str.contains('Time')]
+        time_columns = df.columns[df.columns.str.contains('Time')].tolist()
```

Comment:
Why don't simply use 2 conditions like `a.columns.str.contains('Time') & ~a.columns.str.contains('Nanos')`?
Then we don't need additional variable `nanos` and pop operation.